src/server/scripts/Outland/GruulsLair/boss_gruul.cpp

```diff
@@ -114,6 +114,8 @@ class boss_gruul : public CreatureScript
                 case EVENT_HURTFUL_STRIKE:
                     if (Unit* target = SelectTarget(SELECT_TARGET_TOPAGGRO, 1, 5.0f))
                         me->CastSpell(target, SPELL_HURTFUL_STRIKE, false);
+                    else if (me->GetVictim())                                           /*If no target was found for Hurtful Strike, target the MT*/
```

Comment:
Use brackets and // instead of /*

Reply:
I'm not sure what you mean with brackets here, but ive pushed a commit which changes /* to // for all 3 cases.

Reply:
else if (something)
{
Something
}

Reply:
I hope the last commit solves that.